### PR TITLE
Zinc/Solidity Documentation Update

### PR DESCRIPTION
**Pull request by astarinmymind**
1. Added Zinc to zkEVM FAQ.
2. Updated smart contracts section.

### CODE REVIEW

File: docs/zkevm/README.md [hunk 1]
```diff
@@ -161,6 +160,10 @@ For interactions with smart contracts, users will sign an EIP712 message with a
 
 We will be releasing testnet soon! Please sign up on [this form](https://forms.gle/jQQnJJeuVSVcmkqj9).
 
+### What is the status of Zinc?
+
+We have paused [Zinc](https://zksync.io/dev/contracts/#zinc) development in favor of a [Solidity-first approach](https://medium.com/matter-labs/unisync-a-port-of-uniswap-v2-on-the-zkevm-b12954748504). We will resume work on Zinc after Solidity is released!
```
Comment (popzxc) [inline docs/zkevm/README.md]: "We have paused" sounds pretty radical. Maybe choose somewhat softer phrasing, e.g. "For now, our main priority is a Solidity-first approach. As soon as the Solidity compiler is released and stable, we'll focus on bringing Zinc to zkSync 2.0"?

File: docs/dev/contracts/README.md [hunk 2]
```diff
@@ -33,87 +33,56 @@ circuit per program. A single circuit can be used instead, which needs to be aud
...
+## Solidity
 
-Zinc is created to fill the gap between these two worlds by providing a simple, reliable smart contract language that is optimized for ZKP circuits and is easy for
-developers to learn.
+### Porting Smart Contracts
 
-The framework includes a simple, Turing complete, safety-focused, general-purpose language designed specifically for
-developing smart contracts and zero-knowledge proof circuits with a flat learning curve. The syntax and semantics closely
-follow that of [Rust](https://www.rust-lang.org/).
+Most DeFi and NFT projects will work with no code changes. However, in the first version, calls to SHA256 and Keccak256 will be replaced with a circuit friendly hash function automatically by the compiler. A few other cryptographic primitives are currently also unsupported, for example ecrecover and the cryptographic precompiles.
```
Comment (popzxc) [inline docs/dev/contracts/README.md:40]: > However, in the first version, calls to SHA256 and Keccak256 will be replaced with a circuit friendly hash function automatically by the compiler. 

Is it still correct? IIRC, we decided to provide precompiles even for the testnet.

File: docs/dev/contracts/README.md [hunk 3]
```diff
@@ -33,87 +33,56 @@ circuit per program. A single circuit can be used instead, which needs to be aud
...
 
-The Zinc programming language has an official [book](https://zinc.zksync.io/), which provides an introduction to its core
-principles and conventions.
 
-### Getting help
+* For _read_ requests: any web3-compliant framework in any language will work out of the box, with additional optional zkSync L2 specific functionality.
+* For _write_ requests (sending transactions): due to fundamental differences between L1 and L2, you will have to write some additional code (for example, zkSync supports paying fees in any token, so sending a transaction will involve choosing a token to pay fees).
 
-You can ask questions and get assistance in our [Discord](https://discord.gg/5b6s7VTC) developer chat room.
+You can reuse your current frontend with only minimal changes (sending transactions is different).
```
Comment (popzxc) [inline docs/dev/contracts/README.md:51]: Is this still correct? We decided to implement full support for ECDSA transactions.
I'd say "you may need", but not "you will have to" anymore.

File: docs/dev/contracts/README.md [hunk 4]
```diff
@@ -33,87 +33,56 @@ circuit per program. A single circuit can be used instead, which needs to be aud
...
 
-If you would like to migrate an existing project to [zkSync](https://zksync.io) and require help, please send us an
-email at hello@matter-labs.io.
 
-## Solidity
+### Submitting Transactions
 
-[Solidity](https://docs.soliditylang.org/en/v0.8.1/) is a general-purpose language with a huge codebase and
-numberous DeFi projects, a language adopted by thousands of blockchain developers from all over the world.
+For interactions with smart contracts, users will sign an EIP712 message with a hash of the calldata. Since EIP712 is based on a native Ethereum signature, all wallets, even hardware wallets, will work without any extensions required.
```
Comment (popzxc) [inline docs/dev/contracts/README.md:56]: Also not the case.

**Review by StanislavBreadless — APPROVED** [review]

File: docs/dev/contracts/README.md (hunk 3 above)
Comment (barakshani) [inline docs/dev/contracts/README.md:51]: I believe it would be quite helpful to specify the required changes.

File: docs/dev/contracts/README.md [hunk 5]
```diff
@@ -33,87 +33,56 @@ circuit per program. A single circuit can be used instead, which needs to be aud
...
+developing smart contracts and zero-knowledge proof circuits with a flat learning curve. The syntax and semantics closely
+follow that of [Rust](https://www.rust-lang.org/).
 
-Also, feel free to test arbitrary smart contracts using our Zandbox server.
+The Zinc compiler uses LLVM as its middle-end and back-end, which provides a powerful set of solutions for
+code optimization.
 
-[This tutorial](https://zinc.zksync.io/07-smart-contracts/02-minimal-example.html) explains how to deploy a smart
-contract to the Rinkeby testnet.
+**We are currently fully focused on a [Solidity-first approach](https://medium.com/matter-labs/unisync-a-port-of-uniswap-v2-on-the-zkevm-b12954748504). We will resume work on Zinc after Solidity is released!**
```
Comment (barakshani) [inline docs/dev/contracts/README.md:82]: "We will resume work": @astarinmymind is this grammatically correct?

File: docs/dev/contracts/README.md (hunk 3 above)
Comment (StanislavBreadless) [reply docs/dev/contracts/README.md:51]: They may potentially change, so better not to

File: docs/dev/contracts/README.md (hunk 5 above)
Comment (astarinmymind) [reply docs/dev/contracts/README.md:82]: yes

**Pull request merged**